src/plugins/analytics.js

```diff
@@ -0,0 +1,54 @@
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment'
+
+export default (function () {
+  if (!ExecutionEnvironment.canUseDOM) {
+    return null
+  }
+
+  // Emit events for all github link clicks
+  function anchorListener(event) {
+    let element = event.target
+    if (
+      element.tagName == 'A' &&
+      element.href.startsWith('https://github.com')
```

Comment:
Will this ever link to a subdomain like gh pages?

Reply:
No I think all github pages are at github.io and they have subdomains